docs/release/release_0_3_0.md

```diff
@@ -113,14 +115,14 @@ case, we provide an equivalent API for the same functionality.
   to any layer type.
 - `add_image(path=...)` and `add_labels(path=...)` have been removed. Users
   should use `viewer.open(...)` instead.
```

Comment:
@sofroniewn what is the syntax to ensure a labels layer is returned? `viewer.open(('image.tif', 'Labels'))`?

Reply:
`viewer.open('image.tif', layer_type='labels')`

Reply:
I can add this

Reply:
`viewer.open('image.tif', layer_type='labels')`

Reply:
added more info here